packages/openapi-framework/test/constructor.ts

```diff
@@ -181,5 +181,10 @@ describe('OpenapiFramework', () => {
         });
       });
     });
+
+    it('should accept console as a logger', () => {
+      options.logger = console;
+      new OpenapiFramework(options);
```

Comment:
instead of dummyLogger instead of the default, can we use console?

Reply:
sure